mlrun/projects/pipelines.py

```diff
@@ -291,15 +299,18 @@ def _create_enriched_mlrun_workflow(
                 for function_obj in functions:
                     # we condition within each function since the comparison between the function and
                     # the kfp pod may change depending on the attribute type.
-                    try:
-                        _set_priority_class_name_on_kfp_pod(
-                            kfp_step_template, function_obj
-                        )
-                    except Exception as err:
-                        kfp_pod_name = kfp_step_template.get("name")
-                        logger.warning(
-                            f"Unable to enrich kfp pod {kfp_pod_name}", error=str(err)
-                        )
```

Comment:
I think this shouldn't be removed, main motivation around that try except was to not fail the whole workflow as a result of failed enrichment for one function.